tools/driver/p4c_src/driver.py

```diff
@@ -251,7 +251,32 @@ def runCmd(self, step, cmd):
             return 1
 
         if self._verbose: print('running {}'.format(' '.join(cmd)))
-        p.communicate() # now wait
+        # Wait for the process, if we get CTRL+C during that time, forward it
+        # to the process and continue waiting (leave the program to resolve
+        # it), if we get other error kill the process:
+        # - prevents unnecessary bactraces in case CTRL+C is pressed
+        # - prevents leaving the child process running of communicate error
+        # - it allows running e.g. debugger from the driver, which is useful
+        #   for development
+        try:
+            while True:
+                try:
+                    p.communicate()
+                    break  # done waiting, process ended
+                except KeyboardInterrupt:
+                    import signal
+                    p.send_signal(signal.SIGINT)
+        except:
+            import traceback
```

Comment:
are local imports considered good style?

Reply:
On the other hand, I don't think there are downsides to moving them to the top (maybe the start could be a tiny bit slower, but likely these modules are loaded somewhere in background anyway). So I moved them to the top.